packages/hash/frontend/src/blocks/page/createPlaceholderPlugin/Placeholder.tsx

```diff
@@ -0,0 +1,27 @@
+import { Box, styled, Typography } from "@mui/material";
+
+const Command = styled(Box)(({ theme }) => ({
+  color: theme.palette.gray[50],
+  borderRadius: 3,
+  background: theme.palette.gray[10],
+  width: "1.875em",
+  height: "1.875em",
+  display: "flex",
+  alignItems: "center",
+  justifyContent: "center",
+}));
+
+const Container = styled(Typography)(({ theme }) => ({
+  position: "absolute",
```

Comment:
What's this do? Have you checked this cross browser? `position: "absolute"` without `top`/`left` behaves differently in different browsers.

Reply:
Checked on all browsers, all looked fine, rendered at top-left. But to make sure I can add `top`/`left`, makes sense. It can act differently in future without us noticing it when the parent styling changes

Reply:
Updated here https://github.com/hashintel/hash/pull/953/commits/f44d000dcca0473c6d88298156600f2f268c47bf